pkg/db/db.go

```diff
@@ -105,7 +105,7 @@ func (c Client) NeedsUpdate(ctx context.Context, cliVersion string, light, skip
 	}
 
 	if db.SchemaVersion == metadata.Version && metadata.Type == dbType &&
-		c.clock.Now().Before(metadata.NextUpdate) {
+		c.clock.Now().Before(*(metadata.NextUpdate)) {
```

Comment:
If metadata.NextUpdate is nil, it causes panic.

Reply:
Yeah like we discussed it has been switched now: https://github.com/aquasecurity/trivy-db/commit/64db1801514c32bb0a0413bbb9cc45d2f2a36969